assets/scss/btn.scss

```diff
@@ -0,0 +1,46 @@
+.btn {
+  border: 1px solid #ccc;
```

Comment:
We may want to consider a `font-size` of `19px` and `font-weight` of `500` similar to the marketing site in order to meet AA guidelines with the GeoIP green and MM medium blue color.